demo/Snakefile

```diff
@@ -44,14 +45,16 @@ config["lcr-modules"]["_shared"]["samples"] = SAMPLES
 
 
 # Load module-specific snakefiles
-include: "../modules/star/1.1/star.smk"
-include: "../modules/manta/2.1/manta.smk"
+#include: "../modules/star/1.1/star.smk"
+#include: "../modules/manta/2.1/manta.smk"
+include: "../modules/liftover/1.0/liftover.smk"
 
 
 ##### TARGETS ######
 
 
 rule all:
     input:
-        rules._manta_all.input,
-        rules._star_all.input
+        #rules._manta_all.input,
+        #rules._star_all.input,
+        rules._liftover_all.input
```

Comment:
If you are adding liftover to the demo you will need to include a demo input file in data or this will fail. Is your plan to include a Sequenza output from the demo exome?

Reply:
I did not add the demo files to be committed to the demo/data directory. I will add files to test both battenberg and sequenza